pages/api/auth/[...nextauth].ts

```diff
@@ -1,8 +1,10 @@
-import { decryptWithAES, getUserById, querySQL } from "@coin-view/api";
+import { getUserById, querySQL } from "@coin-view/api";
 import { NextApiRequest, NextApiResponse } from "next";
 import NextAuth, { NextAuthOptions } from "next-auth";
 import CredentialsProvider from "next-auth/providers/credentials";
 
+const bcrypt = require("bcrypt");
```

Comment:
Use import instead of require to stay consistent

Reply:
Done. The reason I used `require` instead of `import`, is because it was returning an error that it could not find module declaration, that has now been fixed.